team-members.tf

```
@@ -112,4 +112,8 @@ resource "github_team_members" "logius-committer" {
   members {
     username = data.github_user.remy-parzinski.username
   }
+
+  members {
+    username = data.github_user.mrskippy.username
```

```suggestion
    username = data.github_user.jaap-hein-wester.username
```

Het is duidelijk als we zijn echte naam als ID gebruiken

---

Beter kan ik het niet maken. Ik probeerde het al zelf te doen maar ik heb de rechten niet.